src/pages/billing/Project.js

```diff
@@ -57,23 +57,23 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
 
   const tabToTable = {
     workspaces: div({ style: { flexGrow: 1, width: '100%' } }, [
-      _.map(([i, workspace]) => h(WorkspaceCard, { workspace, key: i }), Utils.toIndexPairs(_.map(response => response.workspace, _.filter(response => response.workspace.namespace === projectName, workspaces))))
+      _.map(workspace => h(WorkspaceCard, { workspace, key: workspace.workspaceId }), _.map('workspace', _.filter(response => response.workspace.namespace === projectName, workspaces)))
```

Comment:
This may be a bit cleaner to read, and consolidates the `_.map` functions
```suggestion
      _.flow(
        _.filter(response => response.workspace.namespace === projectName),
        _.map(({ workspace }) => h(WorkspaceCard, { workspace, key: workspace.workspaceId }))
      )(workspaces)
```

Reply:
It occurs to me that I think you can write the filter as 
```js
_.filter({ workspace: { namespace: projectName } })
```

Reply:
I confirmed this filter suggestion works